src/amo/components/SecondaryHero/styles.scss

```diff
@@ -0,0 +1,72 @@
+@import '~amo/css/styles';
+
+.SecondaryHero {
+  color: $black;
+  display: flex;
+  justify-content: space-between;
+  min-height: 216px;
+  padding: $padding-page-l 0;
+}
+
+.SecondaryHero-message {
+  font-size: 16px;
+  line-height: 1.5;
+  margin-top: 16px;
+  width: 25%;
+}
+
+.SecondaryHero-message-headline {
+  font-size: $font-size-l;
+  line-height: 1.333;
+}
+
+.SecondaryHero-message-description {
+  padding: $padding-page 0;
```

Comment:
`$padding-page` is a weird value that crept in from long ago. I recommend avoiding it if you can. Our style guideline is to prefer multiples of 4 so I suggest `8px`. I see `9px` in the mock which is why I chose that.

Additionally, the bottom padding is `20px` in the mocks so I suggest making it `24px` for styleguide consistency.